### PR TITLE
Feature/ELG-10/APIs for Refrigerating appliances

### PR DESCRIPTION
**Pull request by FernandoRamirez-De-Aguilar**
API endpoints for

- Household fridges and freezers
- Wine storage appliances

### CODE REVIEW

File: src/main/java/uk/gov/beis/els/categories/refrigeratingappliances/model/FridgesFreezersForm.java [hunk 1]
```diff
@@ -33,6 +42,7 @@
 
   @FieldPrompt("Total volume of chill and unfrozen compartments in litres (l)")
   @Digits(groups = FridgeGroup.class, integer = 3, fraction = 0, message = "Enter the total volume of chill and unfrozen compartments in litres up to 3 digits long")
```
Comment (dannybetts) [inline src/main/java/uk/gov/beis/els/categories/refrigeratingappliances/model/FridgesFreezersForm.java:44]: Missing some NotNull constraints

**Review by dannybetts — CHANGES_REQUESTED** [review]
Update NotBlank constraints to be NotNull and add missing ones

File: src/main/java/uk/gov/beis/els/categories/refrigeratingappliances/model/FridgesFreezersForm.java (hunk 1 above)
Comment (FernandoRamirez-De-Aguilar) [reply src/main/java/uk/gov/beis/els/categories/refrigeratingappliances/model/FridgesFreezersForm.java:44]: done

**Review by jamesbarnett91 — APPROVED** [review]
LGTM, thanks

**Pull request merged**